### PR TITLE
exo: Add alias

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

**Comment by pierre-emmanuelJ** [comment]
Example

`vm list` cmd
```
$ exo vm list
```
`vm ls`cmd alias
```
$ exo vm ls
```

**Comment by pierre-emmanuelJ** [comment]
remove -> rm

any other ideas ? thanks

**Comment by pierre-emmanuelJ** [comment]
@pyr 
- create -> add
is cool but `firewall` cmd have already an `add` and `create` subcmd, so it's will be not very consistant

File: cmd/exo/cmd/root.go [hunk 1]
```diff
@@ -13,21 +13,24 @@ import (
...
+var gAllAccount *config
 
 //egoscale client
 var cs *egoscale.Client
 
+//Aliases
+var gListAlias = []string{"ls"}
+var gRemoveAlias = []string{"rm"}
+var gDeleteAlias = []string{"del"}
+var gShowAlias = []string{"get"}
```
Comment (greut) [inline cmd/exo/cmd/root.go]: `g` for `global`?

Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/root.go]: yes

**Pull request merged**